README.md

```diff
@@ -157,16 +157,18 @@ Read more:
 - [Scripting](https://github.com/hluk/CopyQ/wiki/Scripting)
 - [Scripting Reference](https://github.com/hluk/CopyQ/blob/master/src/scriptable/README.md)
 
-## Build from Source Code
+## Build from source code
```

Comment:
Can you make all titles in Title Case instead? (E.g. "Build the App") There is even [CopyQ command](https://github.com/hluk/CopyQ/issues/664#issuecomment-285951384) for this (I should add it to repository).